clubs/README.md

```diff
@@ -177,6 +177,12 @@ will come to the front and demo what they built at the end of the meeting.
 A small, but easy way to dramatically improve your first meeting is to bring
 food. Pastries or cookies from a local supermarket are cheap and work well.
 
+Also, to set the tone that the club is a safe space, we recommend citing the
+[code of conduct][code-of-conduct] as well as emphasizing its importance in your
+first meeting.
+
+[code-of-conduct]: ../CONDUCT.md
```

Comment:
Can you use underscores in the reference link label to keep it consistent with our reference link labels elsewhere in this repo (ex. `[code_of_conduct]`)?

What if we also reworded the beginning to remove "safe space"? That term has different meanings to different people and I'm worried that we could easily accidentally offend someone if we set the expectation that our clubs are safe spaces and then don't live up to that in their eyes. I think it's best to avoid using it entirely.

Here's a potential rewording:

> Also, to set the tone and behavior expectations in the club, you should include the [code of conduct][coc] in the introduction presentation and emphasize its importance.
>
> [coc] ../CONDUCT.md